lib/wallet/wallet_dashboard.dart

```diff
@@ -65,7 +64,7 @@ class _WalletDashboardState extends State<WalletDashboard> {
           icon: const Icon(Icons.link))));
     }
 
-    if (bitcoinBalance.total().asSats != 0 && lightningBalance.amount.asSats == 0) {
+    if ((bitcoinBalance.total().asSats > 0 && !channel.isAvailable()) || channel.isInitialising()) {
```

Comment:
❓ `|| channel.isInitialising()` was this condition added so we see the card with the spinner?

Reply:
yes. It can take quite some time, before the channel is actually ready, even if the app already shows a lightning balance. I've added this to indicate to the user, the channel is not ready yet. It's not the nicest design, but it's better than a warning message when trying to open a cfd.